CONTRIBUTING.md

```diff
@@ -426,6 +432,14 @@ To become a Code Owner, open a PR with the following changes:
 
 Be sure to tag the existing Code Owners, if any, within the PR to ensure they receive a notification.
 
+### Emeritus roles
+
+Contributors that are stepping down or reducing their role in the project are encouraged to move to [emeritus](https://github.com/open-telemetry/community/blob/main/guides/contributor/membership.md#emeritus-maintainerapprovertriager). In case of long temporary absences, contributors are encouraged to let maintainers know on the CNCF Slack #otel-collector-dev channel and to mark themselves as 'Busy' on Github.
```

Comment:
> Contributors that are stepping down or reducing their role

Maybe we should say something like "Contributors who are unable to meet the responsibilities of their role". The reason I suggest this is that "reducing their role" could be taken to mean a minor reduction in time spent, which occurs frequently in the normal course of operations for many people.

> on the CNCF Slack #otel-collector-dev channel

I would be in favor of removing this detail for privacy reasons. Maybe offer it as an option but sometimes prolonged absences would be better communicated privately. IMO it's enough that it is communicated to a maintainer, who can then handle further communication if necessary.